File: sunbin/day11/002_문자반복출력하기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/120825

function solution(my_string, n) {
  const strArray = [...my_string];
  let newArray = [];

  strArray.forEach((e) => {
    for (let i = 0; i < n; i++) {
      newArray.push(e);
    };
  })

  const answer = newArray.join("")

  return answer
}

// 다른풀이
function solutionB(my_string, n) {
  const answer = [...my_string].map(v => v.repeat(n)).join("");

  return answer
}

File: sunbin/Lv0/day11/python/002_문자반복출력하기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/120825

def solution(my_string, n):
    new_string = ''
    for i in my_string:
      new_string += i * 3

    return new_string



# 다른 풀이
def solutionB(my_string, n):
    return ''.join(i*n for i in my_string)
